Resources/Prototypes/Research/industrial.yml

```diff
@@ -209,6 +209,7 @@
   - ClothingBackpackHolding
   - ClothingBackpackSatchelHolding
   - ClothingBackpackDuffelHolding
+  - OreBagOfHolding
```

Comment:
considering how niche and expensive this item is, you can probably just put it in the salvage equipment tech rather than Bluespace Storage.